platform/mbed_wait_api.h

```diff
@@ -46,54 +46,14 @@ extern "C" {
  * int main() {
  *     while (1) {
  *         heartbeat = 1;
- *         wait(0.5);
+ *         wait_us(500);
  *         heartbeat = 0;
- *         wait(0.5);
+ *         wait_us(500);
```

Comment:
same here

Reply:
@maciejbocianski We try to showcase time precision sleeping, but we are working better example to showcase wait API.